### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by sushilkm**
Added tests for:
- [ ] pkg/version/versions.go
- [ ] pkg/repo/repo.go

### CODE REVIEW

**Review by technosophos — APPROVED** [review]
This all looks great. I'm LGTMing it, but have to wait until we fix CircleCI to merge it.

**Review by bacongobbler — APPROVED** [review]
more tests is always a 👍 in my books.

**Comment by technosophos** [comment]
@sushilkm Would you mind rebasing and pushing again? We made a change to the circle config file to fix the CI failure issue. Sorry for the inconvenience. And thanks again!

**Pull request merged**
